src/devices/Display/SevenSegment/ISevenSegmentDisplay.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Iot.Device.Display.SevenSegment.Enums;
```

Comment:
since you're trying to generalize this I'd rather create `SevenSegmentDisplay` binding or just `Ht16k33`